Makefile

```diff
@@ -6,7 +6,7 @@ GC = extalloc # (extalloc, extalloc_leaking)
 VERSION = 0.31.0-dev
 IMAGE = tinygo/${TARGET}
 
-WASMOPT_PATH = tinygo/lib/binaryen/bin/wasm-opt
+WASMOPT_PATH = /tinygo/lib/binaryen/bin/wasm-opt
```

Comment:
nit: unnecessary change

Reply:
I was not able to run `make build-docker-release `without it. The error was: `cannot use "tinygo/lib/binaryen/bin/wasm-opt" as wasm-opt (from WASMOPT environment variable): fork/exec tinygo/lib/binaryen/bin/wasm-opt: no such file or directoryBuild - tinygo version: 0.31.0-dev, gc: extalloc  (no debug)`